pegasus/rake/generate_pdfs.rake

```diff
@@ -18,23 +19,57 @@ def pdf_conversions_for_files(file_pattern, url_extension)
   end
 end
 
+def pdf_conversions_for_state_pages(state_codes)
+  [].tap do |state_conversion_infos|
+    state_codes.each do |state_code|
+      file_path_without_makepdf_extension = sites_v3_dir('code.org/public/advocacy/state-facts/') + state_code
+      file_path_without_extension = file_path_without_makepdf_extension
```

Comment:
I don't understand why you need both file_path_without_makepdf_extension and file_path_without_extension?